### ISSUE TITLE
Clean YML files

### ISSUE THREAD

**Issue by apulsipher** [EpiForeSITE/software#20]
We need to clean the YML files.

For example, there is a `reviewer` field that is legacy.

**Comment by apulsipher** [comment]
@gvegayon Can you look at the YML template in [`data/README.md`](https://github.com/EpiForeSITE/software/blob/main/data/README.md) and list here which YML key-values can be removed? Also, if you have any other cleaning in mind for this, let me know.

**Comment by gvegayon** [comment]
This should be removed for sure:

```

reviewer:
github_repo_new_or_old_if_existing_one:
complete_yes_no:
pkg_dev_assessment_how_hard_is_to_make_into_a_package_notes:
overall_assessment_easy_win_needs_some_work_needs_lots_of_work_long_term_project:
```

But be mindful about the github repo entry. I don't remember which one I was using to generate the README.md file.

**Comment by gvegayon** [comment]
Also, the entry names are cryptic, so if you can come up with easier names, that'd be awesome too!

**Comment by apulsipher** [comment]
> Also, the entry names are cryptic, so if you can come up with easier names, that'd be awesome too!

I updated the entry names in #23. Let me know what you think of those changes.

**Comment by apulsipher** [comment]
Resolved in #25
